### ISSUE TITLE
pad_observations_v0 not work

### ISSUE THREAD

**Issue by P90-RushB** [PettingZoo-Team/SuperSuit#102]
i found a bug.

// now, obs of agent is (8,), adversiary_0 is (19,)
env = simple_push_v2.parallel_env(max_cycles=50)

// after this func, the observation_space of both is box(19), but ...
env = ss.pad_observations_v0(env)

// obs['agent_0'].shape is (8,)
obs = env.reset()

**Comment by benblack769** [comment]
I was able to reproduce this issue. Thanks for bringing it up! I'll make sure to fix it asap:

```
import supersuit as ss
env = simple_push_v2.parallel_env(max_cycles=50)

env = ss.pad_observations_v0(env)

obs = env.reset()
print(obs[env.agents[0]].shape)
print(obs[env.agents[1]].shape)
```

**Comment by benblack769** [comment]
This problem should be solved with #103

**Comment by benblack769** [comment]
The patch has been released in supersuit 3.1.0. Please let us know if you have any further issues.
